Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -948,6 +948,18 @@ void vtkSlicerVolumeRenderingLogic::UpdateDisplayNodeFromVolumeNode(
 }
 
 //----------------------------------------------------------------------------
+void vtkSlicerVolumeRenderingLogic
+::UpdateDisplayNodeFromVolumeNode(vtkMRMLVolumeRenderingDisplayNode *paramNode,
+                                  vtkMRMLVolumeNode *volumeNode)
+{
+  vtkMRMLVolumePropertyNode *propNode =
+    paramNode ? paramNode->GetVolumePropertyNode() : NULL;
```

Comment:
Shouldn't we return with an error if paramNode is NULL? (log an error and/or return with a bool value)